backend/metamodel/syson-sysml-metamodel/src/main/java/org/eclipse/syson/sysml/impl/NamespaceImpl.java

```diff
@@ -257,9 +261,9 @@ public Membership resolve(String qualifiedName) {
         if (qualification == null) {
             result = resolveLocal(name);
         } else {
-            Membership namespaceMembership = resolve(qualification);
-            if (namespaceMembership != null && (namespaceMembership instanceof Namespace)) {
-                result = ((Namespace) namespaceMembership).resolveVisible(name);
+            Element namespace = resolve(qualification).getMemberElement();
```

Comment:
I think you should handle the case when resolve return null

Reply:
Done